### ISSUE TITLE
Field list in Mukurtu

### ISSUE THREAD

**Issue by HughP** [MukurtuCMS/mukurtu-shared-xsl#2]
Greetings,

Is there a list of fields or data model specifically in Mukurtu? I see mappings to different schemas like MARC. I'm interested in metadata reuse for resource discovery purposes. I'm at the University of North Texas working on metadata interoperability for collections with coverage of indigenous people related content. Maybe we can Zoom? https://hughandbecky.us/Hugh-CV/

**Comment by taylor-steve** [comment]
Hello. There isn't a specific list of fields/metadata map in the Mukurtu CMS software, but one might exist in the broader context of [Mukurtu](https://mukurtu.org/). I will pass your query on to the people who would know. They might also have comments on what I've written below and if we want to Zoom.

Mukurtu CMS (version 3) is built on top of Drupal 7. The initial content types/metadata schemas are essentially a template implementation of the broader Mukurtu concepts. They are not static, users can add/modify/remove fields and content types to fit their specific needs. Any given site running Mukurtu CMS may have site specific metadata schemas/content modeling. For interoperability purposes we might need to discuss if you are targeting stock Mukurtu CMS sites or _any_ Mukurtu CMS site, in which case it's more a question of how you'd target an arbitrary Drupal 7 site.

**Comment by taylor-steve** [comment]
Whoops, I just saw this was in the context of the Mukurtu shared XSL repo and not the CMS project as a whole :) This is a good example of what I was discussing. These tools make the assumption that the site is stock Mukurtu.

**Comment by alexmerrill** [comment]
@HughP   Thanks for the question.  I would be happy to talk about interoperability.  As Steve notes the internal data model is pretty flexible and individual sites can amend and adapt the model for their uses.  We do map known/stock metadata fields to Dublin Core on export to XML and can share a draft Metadata Application Profile for that import/export.  if you would like we can set-up a time to Zoom to discuss this and your wider project.

**Comment by HughP** [comment]
@alexmerrill  Yes please Let's Zoom. I'm in Eugene, Oregon. So I think we are in the same time zone. email < i AT hp3 DOT me > We can set up a time, I'm interested in any documentation or a link to the export schema you have. I have worked with D7 before and am well aware of the "flexibility" it offers.
